e2e/test/iothub/service/PurgeMessageQueueE2ETests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.E2ETests.iothub.service
+{
+    /// <summary>
+    /// E2E test class for PurgeMesageQueue.
+    /// </summary>
+    [TestClass]
+    [TestCategory("E2E")]
+    [TestCategory("IoTHub")]
+    public class PurgeMesageQueueE2ETests : E2EMsTestBase
+    {
+        [TestMethod]
+        public async Task PurgeMessageQueueOperation()
+        {
+            Message testMessage = ComposeD2CTestMessage();
+            using var sc = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionString);
+            var deviceId = TestConfiguration.IoTHub.X509ChainDeviceName;
+            var expectedResult = new PurgeMessageQueueResult()
```

Comment:
nit: when using an initializer list and no ctor parameters, you can drop the parentheses.
```suggestion
            var expectedResult = new PurgeMessageQueueResult
```